### DISCUSSION TITLE
Wheel Selector

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#9452] [CLI - Capacitor / Cordova modes]
I have a select with 60 items in it, however that is not ideal on mobile. I was wondering how users design with issues like this when normally mobile app selectors use a wheel type of selector. Is there an option or way of having this in quasar?

![image](https://user-images.githubusercontent.com/1416749/119884481-aee2c580-befe-11eb-86ba-09ed004fd906.png)

**Comment by ontwikkelfabriek** [comment]
@JokerMartini 
Quasar doesn't have this component or anything similar. However I found this Vue Wheel component:

demo:
https://vue-scroll-picker.dist.be/

repo:
https://github.com/wan2land/vue-scroll-picker 

Please keep us posted on your findings with this or other ways to implement/integrate a spinning wheel input in Quasar Apps!

**Reply by hawkeye64** [reply]
https://github.com/quasarframework/quasar-ui-qscroller

![image](https://user-images.githubusercontent.com/10262924/120077000-73164000-c065-11eb-8904-06669e9294ca.png)

**Reply by hawkeye64** [reply]
You can scroll anything

![image](https://user-images.githubusercontent.com/10262924/120077029-8fb27800-c065-11eb-976c-2fc8be47412d.png)
